frontend/src/pages/Landing/index.css

```diff
@@ -250,14 +250,10 @@ a.desc {
   background: #c5c5c5;
 }
 
-#caughtUp {
+.smart-hub--caught-up {
   padding: 100px 320px;
   white-space: nowrap;
-}
-
-#caughtUp h2 {
-  white-space: nowrap;
-  padding-left: 50px;
+  text-align: center;
 }
 
 #beginNew {
```

Comment:
This ```#beginNew``` can probably also just be replaced with a helper class on whichever element it applies to

Reply:
CSS cleaned up and helper class applied.